src/panels/lovelace/types.ts

```diff
@@ -6,3 +8,14 @@ export interface LovelaceCard {
   getCardSize(): number;
   setConfig(config: LovelaceConfig): void;
 }
+
+export interface EntityConfig {
+  entity: string;
+  name: string;
+  icon: string;
+}
+
+export interface EntityRow {
```

Comment:
I think that we should make a `entity-rows/types.ts` file and move them there.

Reply:
So all cards Entity Rows will be stored there or just a basic EntityRow that all files extend?

Reply:
Just the basic one, like we have a basic lovelace card. Only the thing that we expect on them all.